chatbot/commandcenter/commands/yalldve.py

```diff
@@ -0,0 +1,32 @@
+from ..command import Command
+from ..eventpackage import EventPackage
+import requests
+
+import random
+
+# note: this command only works when run on a machine in same subnet as dot.
+# (this command will work when run on dot.)
+
+class YalldveCommand(Command):
+    def __init__(self):
+        self.name = "$yalldve"
+        self.help = "$yalldve | Slang generator"
+        self.author = "presto"
+        self.last_updated = "October 31st 2019"
+
+    def run(self, event_pack: EventPackage):
+        start=["y","n","ny"]
+        middle=["all","would","could","can","whom","where","don","ain"]
+        end = ["n","t","st","ve"]
```

Comment:
Can these be formatted to [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)? Or even just make them consistent